aliSimple.go

```diff
@@ -96,7 +96,7 @@ func (c *aliClient) getAliClientSche(cred *CredInfo, scheme string) (*aliClient,
 		APIHost: fmt.Sprintf(addrOfAPI, scheme),
 		reqMap: []vKey{
 			{key: "AccessKeyId", val: cred.AccKeyID},
-			{key: "Format", val: "json"},
```

Comment:
https://help.aliyun.com/document_detail/29745.html#:~:text=%E7%9A%84%E7%B1%BB%E5%9E%8B%EF%BC%8C%E6%94%AF%E6%8C%81-,json%E4%B8%8Exml,-%E3%80%82%E9%BB%98%E8%AE%A4%E4%B8%BAXML

Reply:
Could you tell me it is well tested with upper case or lower case? The example of your link was lower case. If yes, that was no problem.

Reply:
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/22412638/166138901-ae496d35-1c67-420f-b7d4-003e8432c26c.png">

According to the documentation, its upper case. And I've tested it works fine. Seems alibabacloud will automate resolve what ever upper case  or lower case